Websites/browserbench.org/index.html

```diff
@@ -39,6 +39,15 @@
 
 <hr>
 
+<a class="benchmark" href="Speedometer3.0" tabindex="0">
```

Comment:
Somehow when running locally this only seems to work with a trailing `/`, I guess that's auto-added by your server?